src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -829,6 +829,14 @@ private function startClass($class, $baseClass, $namespace)
         $bagClass = $this->container->isFrozen() ? 'use Symfony\Component\DependencyInjection\ParameterBag\FrozenParameterBag;' : 'use Symfony\Component\DependencyInjection\ParameterBag\\ParameterBag;';
         $namespaceLine = $namespace ? "namespace $namespace;\n" : '';
 
+        // Inject the parameters as a static property so OPcache can optimize it for speed and memory
+        if ($this->container->getParameterBag()->all()) {
+            $parametersArray = $this->exportParameters($this->container->getParameterBag()->all());
+            $parameters = "\n    private static \$parameters = $parametersArray;";
+        } else {
+            $parameters = '';
```

Comment:
the variable should always be defined (and contain an empty array), as the code still uses it even when the array is empty.